app/build.gradle

```diff
@@ -290,7 +291,7 @@ android {
 
         buildConfigField "String", "HQ_API_USERNAME", "\"${project.ext.HQ_API_USERNAME}\""
         buildConfigField "String", "HQ_API_PASSWORD", "\"${project.ext.HQ_API_PASSWORD}\""
-
+        buildConfigField "String", "FIREBASE_DATABASE_URL", "\"${project.ext.FIREBASE_DATABASE_URL}\""
```

Comment:
how does this feeds into `google-services.json` ?

Reply:
it doesn't, it will come from `~/.gradle/gradle.properties` now

Reply:
Doesn't firebase need it to be in `google-services.json` ?

Reply:
From the tests I ran, it doesn't seems that there aren't services relying on it. I can re-run them just to make sure.

Reply:
can you add what tests you run to the safety story for this PR ?

Reply:
we are adding `FIREBASE_DATABASE_URL` as a build config field but never utilising this field. As such I am struggling to understand the approach behind this change in how Firebase will utilise it from the build config ?

Reply:
I really don't understand the need to move or delete it and a bit hesitant to go ahead with it without testing all Firebase services we use. The url is a public url and is behind authentication, since we already have the `project_id` visible in google services json anyone can anyway construct this url easily with format `https://{project_id}.firebaseio.com`

Reply:
Exactly, that was also my thinking, with the project ID it's easy to put the url together. Is there any particular Firebase we should be concerned about? I can run additional tests.